internal/pkg/agent/cmd/otel_supervised.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-agent/internal/pkg/cli"
+	"github.com/elastic/elastic-agent/internal/pkg/otel"
+	"github.com/elastic/elastic-agent/pkg/core/logger"
+)
+
+func newOtelSupervisedCommandWithArgs(_ []string, streams *cli.IOStreams) *cobra.Command {
```

Comment:
Is this command necessary? It looks like the only meaningful change between it and the normal `otel` command is the config provider. And that can be specified via a command-line argument. Do we want to avoid having this config provider in the normal `otel` command?

Reply:
ok the two commands are now consolidated under the `otel` command (commit 14c87c3e805295850b0d4a19a442e07b8a0edbed) let me know your thoughts 🙂